universal-application-tool-0.0.1/app/modules/MainModule.java

```diff
@@ -14,12 +17,12 @@
  * create modules in other locations by adding `play.modules.enabled` settings to the
  * `application.conf` configuration file.
  */
-public class Module extends AbstractModule {
+public class MainModule extends AbstractModule {
```

Comment:
question: is this ok with Play? For some reason in my head this needed to be called "Module", but looks like that's not true since the tests pass

edit: https://www.playframework.com/documentation/2.8.x/JavaPlayModules#Module-registration - I think I got confused by "will load any class called `Module`" and took that literally - I think this is fine

Reply:
Oh, no, you are right, but ErrorProne doesn't allow default-package classes.  So instead we just specify it directly in `application.conf`.